tests/sentry/seer/anomaly_detection/test_store_data.py

```diff
@@ -30,15 +30,18 @@ def make_event(**kwargs: Any) -> dict[str, Any]:
     return result
 
 
+@freeze_time(before_now(days=2).replace(hour=0, minute=0, second=0, microsecond=0))
```

Comment:
🎉 - any chance you were able to see this go from red to green locally? (ideally, removing this line causes it to be red, then re-adding it makes it 🟢)

Reply:
yep it fails locally w/o this 👍